src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -19,18 +19,18 @@
 
 public class Revolver extends SubsystemBase {
 
-  private final CANSparkMax m_revolver;
+  private final TalonFX m_revolver;
   private int m_drumReps = 0;
+  private double m_setSpeed;
```

Comment:
This is not necessary if you change `m_revolver` to a `WPI_TalonFX`

Reply:
Resolved in commit 4b623d0